default/stringtables/en.txt

```diff
@@ -1402,6 +1402,15 @@ Count of minutes after the cookie record will be considered expired.
 OPTIONS_DB_PUBLISH_STATISTICS
 Enable sending empire staticstics to the player.
 
+OPTIONS_DB_FIRST_TURN_TIME
+Absolute time point in seconds since epoch. If 0 first turn advance will happen after expiring interval.
```

Comment:
This is too jargony. What is "epoch"? What is an expiring interval? Rephrase in terms a user will understand without much context. Also presumably we need to add a GUI widget to set a a time in GMT and the server's local time...

Reply:
Also, put a comma after clauses like "If this is set to 0, the first turn advance will be..."

Reply:
Good question, looks like it is undefined in C++ standard. I'll looks if there better clock which is more determined and compatible with boost::asio timers.
I don't plan to add UI changes in this PR.

Reply:
It sounds like both `std::chrono::system_clock` and `boost::chrono::system_clock` unofficially starts from 01/01/1970 00:00:00 UTC, and `std::chrono::utc_clock` will be introduced only in C++20.
I think I will use `std::chrono::system_clock` and will declare `first-turn-time` to be seconds since 01/01/1970 00:00:00 UTC.

Reply:
Or maybe even `boost::asio::deadline_timer` which is declared to be UTC timer.

Reply:
Rewrite it into `boost::asio::deadline_timer` and made it accepts UTC time in format "2002-01-20 23:59:59". Also I found out it requires fixed interval to be enabled.